src/components/Quote.js

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from 'react';
 
-const Quotes = () => {
+function Quote() {
```

Comment:
Nice job so far😉. Just as required in this milestone, let's try to position our quotes to match the layout in the wireframe😉.

yours
<img width="935" alt="image" src="https://github.com/Katended/math-magicians/assets/64643995/0e45bcdd-9daf-46d7-bdac-82445943ac69">

design
<img width="337" alt="image" src="https://github.com/Katended/math-magicians/assets/64643995/3110c8c1-93e6-40dc-b501-2ad308a42d52">

You can see the design is more obvious and visible because of where it is positioned😉.